bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+OVESETUP_CONFIG/remoteEngineHostSshPort=none:None
+OVESETUP_CONFIG/storageType=none:None
+OSETUP_RPMDISTRO/requireRollback=none:None
+OSETUP_RPMDISTRO/enableUpgrade=none:None
+OVESETUP_DB/upgradeWithHeEl6Hosts=none:None
+OVESETUP_DB/secured=bool:False
+OVESETUP_DB/dumper=str:pg_custom
+OVESETUP_DB/fixDbViolations=none:None
+OVESETUP_DB/filter=none:None
+OVESETUP_DB/restoreJobs=int:2
+OVESETUP_DB/securedHostValidation=bool:False
+OVESETUP_ENGINE_CORE/enable=bool:True
+OVESETUP_CORE/engineStop=none:None
+OVESETUP_SYSTEM/memCheckEnabled=bool:False
+OVESETUP_SYSTEM/nfsConfigEnabled=none:None
+OVESETUP_PKI/renew=none:None
+OVESETUP_CONFIG/isoDomainName=none:None
+OVESETUP_CONFIG/engineHeapMax=str:2048M
+OVESETUP_CONFIG/isoDomainACL=none:None
+OVESETUP_CONFIG/isoDomainMountPoint=none:None
+OVESETUP_CONFIG/engineDbBackupDir=str:/var/lib/ovirt-engine/backups
+OVESETUP_CONFIG/engineHeapMin=str:2048M
+OVESETUP_AIO/configure=none:None
+OVESETUP_AIO/storageDomainName=none:None
+OVESETUP_AIO/storageDomainDir=none:None
+OVESETUP_PROVISIONING/postgresProvisioningEnabled=bool:True
+OVESETUP_APACHE/configureRootRedirection=bool:True
+OVESETUP_APACHE/configureSsl=bool:True
+OVESETUP_VMCONSOLE_PROXY_CONFIG/vmconsoleProxyConfig=bool:True
+OVESETUP_CONFIG/websocketProxyConfig=bool:True
+OVESETUP_CONFIG/fenceKdumpListenerStopNeeded=bool:False
+OVESETUP_APACHE/enable=bool:False
+OVESETUP_CONFIG/engineServiceStopNeeded=bool:False
+OVESETUP_ASYNC/clearTasks=bool:False
+OVESETUP_DWH_CORE/enable=bool:False
+OVESETUP_DWH_PROVISIONING/postgresProvisioningEnabled=bool:False
+OVESETUP_CONFIG/imageioProxyConfig=bool:False
+# Don't install the the OVN provider:
+OVESETUP_OVN/ovirtProviderOvn=bool:False
+# Don't perform database full vacuum:
+OVESETUP_DB/engineVacuumFull=bool:False
+EOF
+
+/usr/bin/engine-setup --config=/answers.conf --offline --accept-defaults
+runuser -u postgres -- psql engine -c "UPDATE vdc_options set option_value = 'false' where option_name = 'InstallVds';"
+runuser -u postgres -- psql engine -c "UPDATE vdc_options set option_value = 'false' where option_name = 'EncryptHostCommunication';"
+runuser -u postgres -- psql engine -c "UPDATE vdc_options set option_value = 'true' where option_name = 'UseHostNameIdentifier';"
+/usr/bin/systemctl restart ovirt-engine
+
+# Setup a NFS server for SD
+mkdir -m777 /nfsexport
+echo "/nfsexport *(rw,sync)" > /etc/exports
+/usr/bin/systemctl enable --now nfs-server
+
+# Install VDSM
+rpm --import https://raw.githubusercontent.com/CentOS-Storage-SIG/centos-release-storage-common/master/RPM-GPG-KEY-CentOS-SIG-Storage
+yum install -y vdsm vdsm-client vdsm-hook-faqemu
+cat << 'EOF' > /etc/vdsm/vdsm.conf
+[vars]
+ssl = false
+fake_kvm_support = true
+EOF
+# WA until commit bf0f31e2b
+sed -i "s/pc-i440fx-rhel7.2.0/pc-i440fx-rhel7.3.0/g" /usr/libexec/vdsm/hooks/after_get_caps/10_faqemu
+sed -i "/def _graphics_spice_to_vnc(/i @workaround()" /usr/libexec/vdsm/hooks/before_vm_start/10_faqemu
```

Comment:
Changing files in `/usr`? That seems wrong, these are under the purview of rpm packages.